server/document.js

```diff
@@ -106,9 +99,10 @@ export class Head extends Component {
 
     return <head {...this.props}>
       {(head || []).map((h, i) => React.cloneElement(h, { key: h.key || i }))}
-      {page !== '/_error' && <link rel='preload' href={`${assetPrefix}/_next/static/${buildId}/pages${pagePathname}`} as='script' nonce={this.props.nonce} />}
-      <link rel='preload' href={`${assetPrefix}/_next/static/${buildId}/pages/_app.js`} as='script' nonce={this.props.nonce} />
-      <link rel='preload' href={`${assetPrefix}/_next/static/${buildId}/pages/_error.js`} as='script' nonce={this.props.nonce} />
+      <link rel='preload' href={`${assetPrefix}/_next/static/runtime/bootstrap.js`} as='script' />
```

Comment:
Is there an agreement about a further script request, or can the bootstrap code be integrated into a bundle?

Reply:
There's no guarantee about execution order of scripts, so it would be hard to ensure that the functions in `bootstrap.js` are run first. I looked into adding it into the webpack runtime, but it seemed more complicated than it was worth.